main.tf

```diff
@@ -19,6 +20,20 @@ resource "aws_customer_gateway" "default" {
   tags       = module.this.tags
 }
 
+module "logs" {
+  source  = "cloudposse/cloudwatch-logs/aws"
+  version = "0.6.7"
+
+  attributes = ["vpn"]
+
```

Comment:
```suggestion

```

Reply:
Removing this as it doesn't really help clarify and I believe is unnecessary for the overall logging to function. Folks can pass down additional attributes via context if needed.